### PR TITLE
Fix & re-enable General-anzeiger-bonn.de.xml

### PR DESCRIPTION
**Pull request by rmueller83**

### CODE REVIEW

File: src/chrome/content/rules/General-anzeiger-bonn.de.xml [hunk 1]
```diff
@@ -1,14 +1,8 @@
...
-Fetch error: http://epaper.general-anzeiger-bonn.de/ => https://epaper.general-anzeiger-bonn.de/: (51, "SSL: no alternative certificate subject name matches target host name 'epaper.general-anzeiger-bonn.de'")
-
--->
-<ruleset name="General-anzeiger-bonn.de" default_off='failed ruleset test'>
+<ruleset name="General-anzeiger-bonn.de">
 	<target host="general-anzeiger-bonn.de" />
 	<target host="www.general-anzeiger-bonn.de" />
 	<target host="trauer.general-anzeiger-bonn.de" />
-	<target host="epaper.general-anzeiger-bonn.de" />
+	<target host="epaper.ga-bonn.de" />
```
Comment (J0WI) [inline src/chrome/content/rules/General-anzeiger-bonn.de.xml]: You should add a redirect from `epaper.general-anzeiger-bonn.de` to `epaper.ga-bonn.de` and a comment about the issue.
Please try to add all hosts in `*.ga-bonn.de`.

Comment (rmueller83) [reply src/chrome/content/rules/General-anzeiger-bonn.de.xml]: Why do I need to add a redirect? This is already done on the server side. If I open http://epaper.general-anzeiger-bonn.de/ i am redirected to https://epaper.ga-bonn.de/

Comment (J0WI) [reply src/chrome/content/rules/General-anzeiger-bonn.de.xml]: The redirect is made though unsecured http, so you could hijack it. We want HTTPS Everywhere.

**Comment by rmueller83** [comment]
Ok, I understand. So I added the redirect.

**Pull request merged**